webpack/assets/javascripts/react_app/components/HostDetails/Properties/__snapshots__/Properties.test.js.snap

```diff
@@ -0,0 +1,237 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Considering these will all likely change many times, does it make sense to include snapshots right now?